src/utils/api.js

```diff
@@ -10,4 +10,9 @@ export const getArticles = async () => {
 export const getArticlesById = async (article_id) => {
     const articleById = await axios.get(`${API_URL}/articles/${article_id}`);
     return articleById
+}
+
+export const getCommentsByArticle = async (article_id) => {
+  const CommentsByArticle = await axios.get(`${API_URL}/articles/${article_id}/comments`)
+  return CommentsByArticle
 }
```

Comment:
Consider refactoring these api requests to use axios instances instead, which can be found here in the notes:
https://notes.northcoders.com/courses/js-front-end/axios#axios-instance